projects/robot_arm/test_robot_arm.py

```diff
@@ -19,22 +19,13 @@
 #  http://numenta.org/licenses/
 #
 # ------------------------------------------------------------------------------
-import gym
```

Comment:
Removed all imports that weren't being used, and sorted the remaining according to PEP8 rules. That is done with isort, which is also an autoformatter specific for imports.